tape/src/main/java/com/squareup/tape/TapeLogger.java

```diff
@@ -0,0 +1,8 @@
+package com.squareup.tape;
+
+public interface TapeLogger {
+
+  public void log(String message);
+
+  public static final TapeLogger NO_LOG = (message) -> {};
```

Comment:
where exactly do we plan to use the tape logger? It seemed like we only wanted either throwing, or ignored.

Reply:
The idea would be for queues that are not sure they would want to adopt this behaviour. They could turn on a log and then take a look for a few weeks to see what the behaviour would be like.

Reply:
see my comment above

also you can remove 'public', 'static', and 'final' from all members inside an interface. those are the default.